test/integration/scheduler/priorities_test.go

```diff
@@ -361,3 +362,105 @@ func TestEvenPodsSpreadPriority(t *testing.T) {
 		})
 	}
 }
+
+// TestDefaultPodTopologySpreadScore verifies that the PodTopologySpread Score plugin
+// with the system default spreading spreads Pods belonging to a Service.
+// The setup has 500 nodes over 3 zones.
+func TestDefaultPodTopologySpreadScore(t *testing.T) {
+	testCtx := initTestSchedulerForPriorityTest(t, podtopologyspread.Name)
+	t.Cleanup(func() {
+		testutils.CleanupTest(t, testCtx)
+	})
+	cs := testCtx.ClientSet
+	ns := testCtx.NS.Name
+
+	var nodes []*v1.Node
+	for i := 0; i < 500; i++ {
+		nodeName := fmt.Sprintf("node-%d", i)
+		zone := fmt.Sprintf("zone-%d", i%3)
+		node, err := createNode(cs, st.MakeNode().Name(nodeName).Label(v1.LabelHostname, nodeName).Label(v1.LabelZoneFailureDomainStable, zone).Obj())
+		if err != nil {
+			t.Fatalf("Cannot create node: %v", err)
+		}
+		nodes = append(nodes, node)
```

Comment:
it seems the node slice is not used anywhere.

Reply:
I meant to delete them, but `testutilis.CleanupTest` removes the nodes and namespace, so actually there is not need to remove Pods and Service manually.